src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
```

Comment:
1. Constants should be uppercased 
2. Constants shouldn't be placed inside function, you have to move them on the global level
```suggestion
  const PRICE = 40;
  const SMALL_DISCOUNT = 20;
  const BIG_DISCOUNT = 50;
```

Reply:
done